jetty-runner/src/main/java/org/eclipse/jetty/runner/Runner.java

```diff
@@ -78,7 +78,9 @@
             org.eclipse.jetty.plus.webapp.EnvConfiguration.class.getCanonicalName(),
             org.eclipse.jetty.plus.webapp.PlusConfiguration.class.getCanonicalName(),
             org.eclipse.jetty.annotations.AnnotationConfiguration.class.getCanonicalName(),
-            org.eclipse.jetty.webapp.JettyWebXmlConfiguration.class.getCanonicalName()
+            org.eclipse.jetty.webapp.JettyWebXmlConfiguration.class.getCanonicalName(),
+            org.eclipse.jetty.webapp.WebAppConfiguration.class.getCanonicalName(),
+            org.eclipse.jetty.webapp.JspConfiguration.class.getCanonicalName()
         };
```

Comment:
Seems like a nice simple change. Is there any way to extend the current integration test to actually test the runner?

Reply:
good point. I need to figure out how to do it

Reply:
@janbartel test added